Formula/semgrep.rb

```diff
@@ -128,6 +112,11 @@ def install
       # Used by semgrep-core for clang to find libtree-sitter.a
       ENV["LIBRARY_PATH"] = lib
 
+      # Officially suggested workaround for breaking change in setuptools v50.0.0
+      # See: https://setuptools.readthedocs.io/en/latest/history.html#v50-0-0
```

Comment:
Can you put the Sourceforge ticket here instead. I care more about the solution than the problem.

Reply:
Done!